be/src/fs/fs.h

```diff
@@ -42,27 +42,32 @@ struct SpaceInfo {
 struct FSOptions {
 private:
     FSOptions(const TBrokerScanRangeParams* scan_range_params, const TExportSink* export_sink,
-              const ResultFileOptions* result_file_options, const TUploadReq* upload, const TDownloadReq* download)
+              const ResultFileOptions* result_file_options, const TUploadReq* upload, const TDownloadReq* download,
+              const THdfsScanNode* hdfs_scan_node)
```

Comment:
Not a good idea to let FSOptions to understand THdfsScanNode, better extract that as AuthInfoOption

Reply:
maybe change it to THdfsProperties is good enough ?